### ISSUE TITLE
Icons don't work properly / no icons list

### ISSUE THREAD

**Issue by oussamaB14** [luisDanielRoviraContreras/flutter_iconsax#7]
icons on the site sound beautiful but they are useless on flutter

**Comment by RalphMarvin** [comment]
Yes very useless on flutter. I've been trying to use it but I had to remove it. No icon list and most icons don't render properly.
Try other packages like the ones below if you've not already found one: 

- heroicons
- flutter_remix
- unicons

They look promising.
